Core/Core/Common/CommonUI/ViewModifiers/Tint.swift

```diff
@@ -34,4 +34,24 @@ extension View {
     public func applyTint() -> some View {
         foregroundStyle(.tint)
     }
+
+    @available(iOS, deprecated: 26, message: "Intended for temporary compatibility before full iOS 26 adoption.")
+    @ViewBuilder
+    public func tintBelow26(_ tint: Color?) -> some View {
+        if #available(iOS 26, *) {
+            self
+        } else {
+            self.tint(tint)
+        }
+    }
+
+    @available(iOS, deprecated: 26, message: "Intended for temporary compatibility before full iOS 26 adoption.")
+    @ViewBuilder
+    public func foregroundStyleBelow26<S>(_ style: S) -> some View where S: ShapeStyle {
```

Comment:
These are used in only in toolbars (navbars)
```suggestion
    public func toolbarItemForegroundStyle<S>(_ style: S) -> some View where S: ShapeStyle {
```

Please rename tintBelow26 similarly.

We may need to unify these button usages eventually, but even until that happens let's "restrict" these with naming.